Foundation/CPURLConnection.j

```diff
@@ -138,6 +143,18 @@ var CPURLConnectionDelegate = nil;
 }
 
 /*
+    Loads the data for a URL request and executes a function on an operation queue when the request completes or fails.
+    @param aRequest contains the URL to obtain data from.
+    @param aQueue The operation queue to which the function is dispatched when the request completes or failed.
+    @param aHandler The function to execute.
+    @discussion If the request completes successfully, the data parameter of the function contains the resource data, and the error parameter is nil. If the request fails, the data parameter is nil and the error parameter contain information about the failure.
+*/
++ (CPURLConnection)sendAsynchronousRequest:(CPURLRequest)aRequest queue:(CPOperationQueue)aQueue completionHandler:(Function)aHandler
```

Comment:
Can we use `usingBlock:` instead of `completionHandler:`. It's more consistent with the other API of Cappuccino